services/QuillLMS/db/migrate/20250307233440_create_classlink_instances.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateClasslinkInstances < ActiveRecord::Migration[7.1]
+  def change
+    create_table :classlink_instances do |t|
+      t.integer :application_id, null: false
+      t.boolean :enabled, null: false
+      t.string :oneroster_application_id, null: false
+      t.string :name, null: false
+      t.string :tenant_name, null: false
+      t.references :district, foreign_key: true, null: false
```

Comment:
Again with the foreign key, though I'll point out that `districts` should be a small enough table that any costs imposed here are very small.

Reply:
Adding to eng discussion